### DISCUSSION TITLE
从个人消息进入视频能否自动删除#reply那段

### DISCUSSION THREAD

**Discussion by harenojou** [the1812/Bilibili-Evolved#3800] [功能建议 / Ideas]
可能因为评论区的bug，带#reply的视频地址是没办法查看热度评论的

**Comment by Geeyun-JY3** [comment]
带 `#reply` 参数本就是为了定位评论而生的。

另外实测可以看热门评论啊，能不能详细说一下怎么看不了热门评论。

**Reply by harenojou** [reply]
我从回复评论或点赞信息里进入的视频评论区只能显示“按时间排序”没有“按热度排序”的选项，刚用github不知道这样子能不能发的出图片
![image](https://user-images.githubusercontent.com/117423382/199886206-8bd855cc-6a63-419f-bb18-5c1bac804dd0.png)

**Reply by Ethkuil** [reply]
我从点赞信息进入评论区，一切正常。没能复现。

**Reply by Ethkuil** [reply]
> 我从点赞信息进入评论区，一切正常。没能复现。

URL是带#reply的。

**Reply by Geeyun-JY3** [reply]
> ![](https://user-images.githubusercontent.com/117423382/199886206-8bd855cc-6a63-419f-bb18-5c1bac804dd0.png)

确实能用图中的 URL 在无痕式窗口复现，但似乎也只是这个视频的评论的定位链接才能复现，其他视频并无法复现。
![image](https://user-images.githubusercontent.com/49396421/199907019-7403ff59-abc9-470f-8819-6964ea6776e9.png)

应该是 b 站 bug。建议反馈给 b 站。

**Reply by Geeyun-JY3** [reply]
今天似乎已无法复现

**Reply by harenojou** [reply]
我也无法复现了，感觉只对消息列表里刚发出去不久的评论有bug，大概过个几天就不会了

**Comment by the1812** [comment]
删了也只是 URL 里看起来删掉了, 该 bug 还是 bug 的
